### PR TITLE
//Manufacturers and suppliers display

### PR DESCRIPTION
**Pull request by maximebiloe**
This is a POC to manage Suppliers and Manufacturers display.
Do you have any suggestions about structure and/or HTML markup ?

Here is the structure suggested for : 
- **List of products**
  capture d ecran 2015-10-08 a 18 43 04
- **Manufacturers and Suppliers aka Brands**
  capture d ecran 2015-10-08 a 18 43 10

Do not hesitate to comment.

### CODE REVIEW

File: themes/StarterTheme/templates/catalog/products.tpl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{if $products|count}
+  <section id="products">
+    <h1>{l s='Products'}</h1>
```
Comment (gskema) [inline themes/StarterTheme/templates/catalog/products.tpl]: This should probably be `h2` or lower

**Comment by gskema** [comment]
I agree on the structure `{product_page}.tpl > products.tpl > product-miniature.tpl`.
As for the product list I believe this would make more sense:

```
product-list.tpl
product-list-item.tpl
```

But then I thought about sliders, where `*-miniature.tpl` makes sense and `*-list-item` - not so much.
Regardless I'd probably stick with `*-list.tpl` and `*-list-item.tpl`.
If the slides differ much from list item, you can always make a new template/extend.

What do others think?

**Comment by maximebiloe** [comment]
Hi @gskema,

Thanks for your remarks.

About the `h1` tag in the `<section id="products">`, we have a Trello card about it (https://trello.com/c/WNUilMDk/151-ensure-all-section-s-have-a-h1), it's still in thinking. So, we take all remarks in consideration.

About the structure, we wanted to be consistent regarding what has been already done about categories but we still listen to others opinions.

File: themes/StarterTheme/templates/catalog/products.tpl (hunk 1 above)
Comment (djfm) [inline themes/StarterTheme/templates/catalog/products.tpl]: I think the h1 is correct, because it is inside a section.

Comment (gskema) [inline themes/StarterTheme/templates/catalog/products.tpl]: There must be only one `h1` in a document, having more than one is a SEO mistake. Sections must have a heading element, but it doesn't have to be `h1` (can be `h2`, `h3`, etc.)

**Comment by inetbiz** [comment]
I have a PR already on structure. Products in a category listing is an itemlist

**Comment by djfm** [comment]
@inetbiz thanks, I'll get to yours right after this one!

**Pull request merged**